I3uckwheat/cards-of-carousal
# Hotfix/update metadata

#### 0. Make sure your issue is linked:

N/A

#### 1. Purpose:

Replace the CRA default title and icon with an appropriate title and icon.


#### 2. Implementation:

Updated the metadata in index.html


#### 3. Possible Issues:

Not really sure if I did this right, but it seems to work.


#### 4. How to Test:

0. Start app backend and frontend
1. Go to app in your web browser
2. Verify that the title and icon for the page are no longer the CRA defaults

#### 5. Cleanup
